Sources/CombineGRPC/Client/DemandBuffer.swift

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019, Vy-Shane Xie
```

Comment:
Truth be told I did rewrite it whilst examining CombineExt's version. I'm not sure how much different these classes would/could be considering they do the same thing. I even preserved some of the comments because they made sense.

CombineExt has an MIT license so if you consider them the authors then you'll need to add the MIT license header and probably note that somewhere in your README (that you have code under different licenses).

If we were keeping CombineExt as a dependency that's probably not a big deal but with the ability to remove CombineExt now it would be better to have everything Apache 2 (like GRPC and SwiftNIO).

Reply:
I personally consider this my re-creation and was assigning Copyright to the project.